packages/sn-filter-pane/src/components/ListboxGrid/distribute-resources.ts

```diff
@@ -1,16 +1,21 @@
 import { IListboxResource } from '../../hooks/types';
-import { IColumn, ISize } from './interfaces';
+import { ExpandProps, IColumn, ISize } from './interfaces';
 
 export const COLLAPSED_HEIGHT = 34;
 export const BUTTON_HEIGHT = 34;
 const BUTTON_SPACING = 8;
 export const ITEM_SPACING = 8;
-const EXPANDED_HEIGHT_LIMIT = 181;
 const COLUMN_MIN_WIDTH = 160;
 const COLUMN_SPACING = 16;
 const EXPANDED_HEADER_HEIGHT = 48;
+const SINGLE_GRID_ROW_HEIGHT = 28;
```

Comment:
Is this referring to the same height as this one: https://github.com/qlik-oss/nebula.js/blob/master/apis/nucleus/src/components/listbox/assets/get-list-sizes/get-list-sizes.js#L36 

If so, we should break them out into a `constants.ts` file and import them in both places.

Reply:
Yes, good point. Will have to to it in another PR.